app/models/article.rb

```diff
@@ -1,2 +1,5 @@
 class Article < ApplicationRecord
+  def author_image
```

Comment:
This needs to handle multiple authors, using the author name as the way to select the image.

Reply:
I've added that in now, also includes a fallback if the image doesn't exist